Helpers.ps1

```diff
@@ -394,12 +394,8 @@ function DeepCopy-Object
         [PSCustomObject] $InputObject
     )
 
-    $memoryStream = New-Object System.IO.MemoryStream
-    $binaryFormatter = New-Object System.Runtime.Serialization.Formatters.Binary.BinaryFormatter
-    $binaryFormatter.Serialize($memoryStream, $InputObject)
-    $memoryStream.Position = 0
-    $DeepCopiedObject = $binaryFormatter.Deserialize($memoryStream)
-    $memoryStream.Close()
+    $serialData = [System.Management.Automation.PSSerializer]::Serialize($InputObject, 64)
+    $DeepCopiedObject = [System.Management.Automation.PSSerializer]::Deserialize($serialData)
 
     return $DeepCopiedObject
```

Comment:
nit: the capitalization is wrong (per the module preferences (local vars should have camelCase)).  This pre-dates your change, but since you're in here, would be ideal to fix it at the same time.

```suggestion
    $deepCopiedObject = [System.Management.Automation.PSSerializer]::Deserialize($serialData)

    return $deepCopiedObject
```

Reply:
Did one better and just returned earlier, got rid of the intermediate variable.